src/wolfkrow/core/tasks/task.py

```diff
@@ -7,17 +7,18 @@
 import collections
 import copy
 import datetime
+import errno
+import json
 import logging
 import os
-import six
```

Comment:
Is six still used anywhere in Wolfkrow? Can we remove this as a dependency in the setup.py file?